<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(75, 75, 75);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="Tab">
      <property name="styleSheet">
       <string notr="true">QTabWidget:pane{
border: none;
}

QTabBar::tab{
color: white;
background-color:  rgb(75, 75, 75);
/*border: 2px solid rgb(0, 0, 0);
padding: 5px 5px;*/
}

QTabBar::tab:selected{
color: orange;
}

QLabel{
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Scoreboard_Tab">
       <attribute name="title">
        <string>Scoreboard</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="4" rowspan="4">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1">
           <widget class="QPushButton" name="AwayScore2Up_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="AwayScore_Label">
            <property name="font">
             <font>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Away_Score</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="AwayScoreDown_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="AwayScoreUp_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
color: white;
border: 1px solid black;
}</string>
            </property>
            <property name="text">
             <string>AWAY</string>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="4">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="horizontalSpacing">
           <number>1</number>
          </property>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="HomeScoreDown_PushButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="HomeScore2Up_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="HomeScoreUp_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="HomeScore_Label">
            <property name="font">
             <font>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Home_Score</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLineEdit" name="lineEdit">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit{
color: white;
border: 1px solid black;
}</string>
            </property>
            <property name="text">
             <string>HOME</string>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2" colspan="2">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QSpinBox" name="spinBox">
            <property name="styleSheet">
             <string notr="true">QSpinBox{
border: none;
color: white;
}

QSpinBox::up-button:pressed{
color: orange;
}</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBox_2">
            <property name="styleSheet">
             <string notr="true">QSpinBox{
border: none;
color: white;
}

QSpinBox::up-arrow:pressed{
color: orange;
}</string>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="Clock_Label">
            <property name="font">
             <font>
              <pointsize>36</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
border: 1px solid black;</string>
            </property>
            <property name="text">
             <string>Clock</string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QSpinBox" name="spinBox_3">
            <property name="styleSheet">
             <string notr="true">QSpinBox{
border: none;
color: white;
}

QSpinBox::up-arrow:pressed{
color: orange;
}</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="spinBox_4">
            <property name="styleSheet">
             <string notr="true">QSpinBox{
border: none;
color: white;
}

QSpinBox::up-arrow:pressed{
color: orange;
}</string>
            </property>
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="Colon2_Label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="Colon_Label">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2" rowspan="3" colspan="2">
         <layout class="QVBoxLayout" name="Period">
          <item>
           <widget class="QPushButton" name="PeriodUp_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Period_Label">
            <property name="font">
             <font>
              <pointsize>28</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
</string>
            </property>
            <property name="text">
             <string>Period</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PeriodDown_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1" rowspan="3" colspan="4">
         <layout class="QHBoxLayout" name="MainButton">
          <item>
           <widget class="QPushButton" name="UpdateTeam_PushButton">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ResetClock_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Clock_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ResetScore_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Swap_PushButton">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Setting_Tab">
       <attribute name="title">
        <string>Setting</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="1" column="1">
               <widget class="QCheckBox" name="checkBox_3">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>.XML</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="Animations_CheckBox">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>.TXT</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QCheckBox" name="checkBox_4">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>JSON</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="styleSheet">
                 <string notr="true">color: white;
font-size: 20px;</string>
                </property>
                <property name="text">
                 <string>Output Setting</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="1" column="2">
               <widget class="QCheckBox" name="checkBox_7">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>Fullscreen</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBox_5">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>Display Current Time</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QCheckBox" name="checkBox_6">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>Current Inputbox</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="styleSheet">
                 <string notr="true">color: white;
font-size: 20px;</string>
                </property>
                <property name="text">
                 <string>Display Setting</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBox_2">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>Milliseconds</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QCheckBox" name="checkBox_8">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>Add zero</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBox_9">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>Hotkey</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">color: white;
font-size: 20px;</string>
                </property>
                <property name="text">
                 <string>Other Setting</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QCheckBox" name="checkBox_10">
                <property name="styleSheet">
                 <string notr="true">QCheckBox{
color: white;
}

/*QCheckBox::indicator{
background-color: #454545;
}

QCheckBox::indicator:checked{
color: orange;
}*/</string>
                </property>
                <property name="text">
                 <string>Always on Top</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="SetHotkey_PushButton">
                <property name="text">
                 <string>Set Hotkey</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="VirtualScoreboard_Tab">
       <attribute name="title">
        <string>Virtual Scoreboard</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Sound_Tab">
       <attribute name="title">
        <string>Sound</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Test Sound</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Play Sound</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Remote_Tab">
       <attribute name="title">
        <string>Remote</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Console:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
